docs/octopus-cloud/index.md

```diff
@@ -21,27 +21,24 @@ You can sign up for Octopus Cloud at [octopus.com/register](https://octopus.com/
 An Octopus account lets you manage your instances of Octopus Cloud.
 
 1. Enter your name.
-1. Provide your email address and click **Create a password**. Please note, these credentials are for you Octopus Account. You will also [create credentials](#create-a-cloud-instance) for your Octopus Cloud instance, when you create it.
-1. On the next screen, provide your company name.
-1. Chose a secure password and enter it twice.
-1. Click **Create my Octopus account**.
+1. Provide your email address and create a password. Please note, these credentials are for your Octopus account and you will use them to log into your Octopus instance through single sign-on.
```

Comment:
>you will use them to log into your Octopus instance through single sign-on

The updated steps here only apply to v2 instances (_not sure how we want to differentiate here_)

Reply:
>_not sure how we want to differentiate here_

I've raised the question here: https://octopusdeploy.slack.com/archives/C44HGJF8D/p1570508665018000. Happy for you to follow through with whatever decision is reached there.